test/suites/perf/scheduling_test.go

```diff
@@ -44,18 +51,81 @@ var _ = Describe("Performance Benchmark", func() {
 					},
 				}})
 			env.ExpectCreated(deployment)
-			start := time.Now()
 			env.ExpectCreated(nodePool, nodeClass)
-			env.EventuallyExpectHealthyPodCount(labelSelector, 100)
-			// Need a way to respond to the last pod healthy event or look at the pod status conditions after the fact to get the exact measurements here.
-			// would also be good to just pull the metrics directly from the Karpenter pod to get the scheduling simulation metrics.
-
-			// env.Monitor.GetLastPodSchedulingEvent()
-			duration := time.Since(start)
+			env.EventuallyExpectHealthyPodCount(labelSelector, replicas)
+		})
+		It("should do simple provisioning and simple drift", func() {
+			deployment := test.Deployment(test.DeploymentOptions{
+				Replicas: int32(replicas),
+				PodOptions: test.PodOptions{
+					ObjectMeta: metav1.ObjectMeta{
+						Labels: testLabels,
+					},
+					ResourceRequirements: v1.ResourceRequirements{
+						Requests: v1.ResourceList{
+							v1.ResourceCPU: resource.MustParse("1"),
+						},
+					},
+				}})
+			env.ExpectCreated(deployment)
+			env.ExpectCreated(nodePool, nodeClass)
+			env.EventuallyExpectHealthyPodCount(labelSelector, replicas)
 
-			fmt.Println("--------- RESULTS ---------")
-			fmt.Printf("This is the duration: %s\n", duration)
+			env.TimeIntervalCollector.Start("Drift")
+			env.TimeIntervalCollector.End("Drift")
```

Comment:
Seems like we probably don't want to call this here. I'm imagining that this was supposed to be a defer statement or should be dropped entirely because we already call it later in test